lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -20,329 +20,378 @@
 options:
   name:
     required: true
+    type: str
     description:
-    - Service name
+      - Service name
   image:
-    type: str
     required: true
+    type: str
     description:
-    - Service image path and tag.
-      Maps docker service IMAGE parameter.
+      - Service image path and tag.
+      - Maps docker service C(IMAGE) parameter.
   resolve_image:
     type: bool
-    required: false
     default: true
     description:
       - If the current image digest should be resolved from registry and updated if changed.
     version_added: 2.8
   state:
     required: true
+    type: str
     default: present
     description:
-    - Service state.
+      - Service state.
     choices:
-    - present
-    - absent
+      - present
+      - absent
   args:
-    required: false
-    default: []
     description:
-    - List comprised of the command and the arguments to be run inside
-    - the container
+      - List arguments to be passed to the container.
   command:
-    required: false
     description:
-    - Command to execute when the container starts.
-      A command may be either a string or a list or a list of strings.
+      - Command to execute when the container starts.
+      - A command may be either a string or a list or a list of strings.
     version_added: 2.8
   constraints:
-    required: false
-    default: []
+    type: list
     description:
-    - List of the service constraints.
-    - Maps docker service --constraint option.
+      - List of the service constraints.
+      - Maps docker service C(--constraint) option.
```

Comment:
Similar here (and all the other ones, I guess):
```suggestion
      - Maps C(docker service create)'s C(--constraint) option.
```
or
```suggestion
      - Corresponds to the C(--constraint) parameter of C(docker service create).
```